src/Url/DrupalGenerator.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace IonBazan\ComposerDiff\Url;
+
+use Composer\Package\CompletePackageInterface;
+use Composer\Package\PackageInterface;
+
+class DrupalGenerator extends GitlabGenerator
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsPackage(PackageInterface $package)
+    {
+        return 'drupal/core' === $package->getName() || in_array($package->getType(), array('drupal-module', 'drupal-theme')) || parent::supportsPackage($package);
```

Comment:
Not sure if having package type `drupal-module` or `drupal-theme` is enough. There are many community modules that are hosted on GitHub with this type so I think this can be simplified to:
```suggestion
        return 'drupal/core' === $package->getName() || parent::supportsPackage($package);
```

Reply:
Ah, I see. Oh, maybe. If there's not test coverage for this, we should add it :)